.github/workflows/drivers_ci.yml

```diff
@@ -0,0 +1,188 @@
+name: Drivers CI Build
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+      - 'release/**'
+      - 'maintainers/**'
+  workflow_dispatch:
+  # we cannot use paths key here since otherwise required_status_checks jobs won't run.
+  # See https://github.com/orgs/community/discussions/26251.
+  # We need to use the paths-filter job.
+
+# Checks if any concurrent jobs under the same pull request or branch are being executed
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  paths-filter:
```

Comment:
We cannot use workflow level `paths` filter because of https://github.com/orgs/community/discussions/26251.
Otherwise our required status checks won't be properly set to "skipped".